### PR TITLE
Suggestion for reviewers

### PR DESCRIPTION
**Pull request by sstevens2**

### CODE REVIEW

File: during.md [hunk 1]
```diff
@@ -64,6 +64,11 @@
   is merged into the lesson as soon as possible.
   Opening draft pull requests is particularly important at the end of a sprint,
   to capture progress up to that point even if a task hasn't yet been completed.
+- **Plan who will review Pull Requests.**
+  Make a plan for who will review each Rull Request.
```
Comment (tobyhodges) [inline during.md]:
```suggestion
  Make a plan for who will review each Pull Request.
```

File: during.md [hunk 2]
```diff
@@ -64,6 +64,11 @@
...
+  As a group you can create list of reviewers by subject so the person who opens the Pull Reqest knows who to is interested and capable of reviewing.
```
Comment (tobyhodges) [inline during.md]:
```suggestion
  As a group you can create list of reviewers by subject so the person who opens the Pull Request knows who is interested and capable of reviewing and can request a review accordingly.
```

File: during.md [hunk 3]
```diff
@@ -64,6 +64,11 @@
...
+  Alternatively, maybe you decide on circle of reviewers. E.g. Aaliyah reviews Juan's PRs, Juan reivews Alex's PRs, and Alex reviews Aaliyah's PRs.
```
Comment (tobyhodges) [inline during.md]:
```suggestion
  Alternatively, you may decide on a circle of reviewers. E.g. Aaliyah reviews Juan's PRs, Juan reviews Alex's PRs, and Alex reviews Aaliyah's PRs.
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
These are fantastic recommendations, @sstevens2. I've suggested a few small typo fixes and one other adjustment.

**Comment by sstevens2** [comment]
Thanks for the review and feedback!

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**